### DISCUSSION TITLE
Voice triggered Panic Button

### DISCUSSION THREAD

**Discussion by jn3va** [Dilbert66/esphome-vistaECP#38] [Ideas]
This is not specific to the VistaECP but has anyone thought about adding a Vista20 zone with a NC relay, and using a voice command to Alexa via HAAS to open the relay and trigger the alarm?  Any pros and cons?   I'm thinking you could create a Zone type 06 or 07 for Burglar panic or type 90 for medical.   Might even use HaaS actionable notifications to have Alexa ask if you really wanted to do this to prevent accidental triggers (Something like.... Did you really want me to do that?  And if you answer No things stop, if no answer or yes they proceed)

@Dilbert66 Thing adding an ESPHome config to trigger the relay(s) the YAML would interfere with anything your code is doing?

**Comment by Dilbert66** [comment]
You can simply use one of the emulated zone expander addresses and trigger one of those zones from HA to do whatever you want.  The only config in the yaml would be to activate one of the expander addresses ton one not physically used in the system.   You can then setup that zone to trigger  a relay, alarm, or whatever you want.
